internal/connector/internal/services/phase/processor_phase.go

```diff
@@ -0,0 +1,112 @@
+package phase
```

Comment:
All statuses, phases and transitions for `processor`'s follow those defined for `connector`'s.